cmd/artifact/install/install.go

```diff
@@ -239,6 +239,13 @@ func (o *artifactInstallOptions) RunArtifactInstall(ctx context.Context, args []
 			destDir = o.rulesfilesDir
 		}
 
+		if _, err = os.Stat(destDir); os.IsNotExist(err) {
+			err = os.MkdirAll(destDir, 0o700)
```

Comment:
The installer (`falcoctl`) and the consumer (`falco`) are two different processes, with different permissions and capabilities. Moreover, the consumer can run into different situations.
Think the syscall use case: Falco runs as root. However, in the plugin's use cases, it may run as non-root.

Considering that, I'm a bit concerned that `0x700` may break some cases.